tasks.py

```diff
@@ -0,0 +1,206 @@
+from invoke import task
```

Comment:
Despite the size of this file, only 4 tasks are defined here. I will leave a brief description of each task.